client/state/themes/actions.js

```diff
@@ -364,6 +368,49 @@ export function clearActivated( siteId ) {
 }
 
 /**
+ * Triggers a network request to install WordPress.com theme on Jetpack site.
+ * After installataion it switches page to the customizer
+ * Requires Jetpack 4.4
+ *
+ * @param  {String}   siteId       Jetpack Site ID
+ * @param  {String}   wpcomThemeId WP.com Theme ID
+ * @return {Function}              Action thunk
+ */
+export function tryAndCustomizeWpcomThemeOnJetpack( themeId, siteId ) {
+	//Add -wpcom suffix. This suffix tells the endpoint that we want to
+	//install WordPress.com theme. Without the suffix endpoint would look
+	//for theme in .org
+	const suffixedThemeId = themeId + '-wpcom';
```

Comment:
I think we can actually keep this function generic -- maybe at some point we really want to install a theme from WP.org; we'll be able to re-use this function then. It's easy enough for the consumer to add the prefix themselves.